tests/providers/test_hostingde.py

```diff
@@ -0,0 +1,26 @@
+# Test for one implementation of the interface
+from lexicon.tests.providers.integration_tests import IntegrationTests
+from unittest import TestCase
+
+# Hook into testing framework by inheriting unittest.TestCase and reuse
+# the tests which *each and every* implementation of the interface must
+# pass, by inheritance from integration_tests.IntegrationTests
+class FooProviderTests(TestCase, IntegrationTests):
+    """Integration tests for hostingde provider"""
+
+    provider_name = 'hostingde'
+    domain = 'eruza.de'
+
+    def _filter_post_data_parameters(self):
+        return ['authToken']
+
+    def _filter_headers(self):
+        return ['Authorization']
+
+    def _filter_query_parameters(self):
+        return ['secret_key']
+
+    def _filter_response(self, response):
```

Comment:
It is not necessary to override this method. Default behavior in superclass `IntegrationTests` is already to return `response` untouched.